syft/pkg/cataloger/internal/pkgtest/test_generic_parser.go

```diff
@@ -44,10 +52,26 @@ type CatalogTester struct {
 	licenseComparer                cmptest.LicenseComparer
 	packageStringer                func(pkg.Package) string
 	customAssertions               []func(t *testing.T, pkgs []pkg.Package, relationships []artifact.Relationship)
+	context                        context.Context
+}
+
+func Context() context.Context {
```

Comment:
Thanks this is great so that we're keeping the tests fast and setting this up one time